### ISSUE TITLE
replication after using copyNamed

### ISSUE THREAD

**Issue by KraemerDEM** [couchbase/couchbase-lite-core#1240]
After using copyNamed to create a copy of a database and starting a pull replicator on the new database,
it seems that couchbaselite unneccessarily still pulls in a lot of data despite the sqlite database already has the latest data.
At least i get a lot of Log messages like 
```
{RevFinder#22} Responded to 'changes' REQ#625 w/request for 0 revs in 0.007453 sec
{RevFinder#22} Received 200 changes (seq '1004081'..'1004316')
```
I think this is because copyNamed  creates new UUIDs which leads to not having a valid checkpoint when opening the new database (which is fine as IMHO that's the whole point of creating new UUIDs here).
However maybe it would be possible to somehow create a checkpoint that "knows" that the local database already has the latest data to avoid this.

**Comment by borrrden** [comment]
This is definitely the intention (see [here](https://github.com/couchbase/couchbase-lite-core/blob/2460fdb3d531c2cb8f617e3fbaf4c3c038b9ce69/Replicator/Checkpointer.cc#L263) ).  So if it is not working the question is "why?"

**Comment by KraemerDEM** [comment]
I've found the reason for this issue:
I've used copyNamed twice, once when exporting the database and then again when importing it again. So on the second call of copyNamed "previousPrivateUUID" will be overridden with a UUID which was never used to create a checkpoint.
So just doing a normal file copy when importing fixes this issue.
Maybe this should be mentioned in the documentation of copyNamed.
